package.json

```diff
@@ -3,13 +3,12 @@
 	"scripts": {
 		"lint": "xo && stylelint source/*.css",
 		"lint-fix": "xo --fix; stylelint --fix source/*.css",
-		"test": "npm run lint && cross-env BABEL_ENV=testing ava && run-s build:minified",
-		"build": "webpack",
-		"build:minified": "cross-env NODE_ENV=production webpack",
-		"watch": "webpack --watch",
+		"test": "npm run lint && cross-env BABEL_ENV=testing ava && run-s build",
+		"build": "webpack --mode production",
+		"watch": "webpack --mode development --watch",
```

Comment:
`--mode development` => `--mode=development`

Reply:
What's the difference?

Reply:
Technically none, just more readable, as it's totally clear `development` is a value of `--mode` and not just an argument to `webpack`.

Reply:
Ugly, but it makes sense 😄